fastestimator/util/google_download_util.py

```diff
@@ -66,3 +68,25 @@ def _download_file_from_google_drive(file_id: str, destination: str) -> None:
                 progress.update(len(chunk))
                 f.write(chunk)
     progress.close()
+
+
+def download_file_from_google_drive(file_id: str, destination: str, max_retries: int = 3) -> None:
+    """Download the data from the Google drive public URL.
+
+    This method will try to download the file for given number of retries till successful.
+
+    Args:
+        file_id: File ID of Google drive URL.
+        destination: Destination path where the data needs to be stored.
+        max_retries: max number of retries.
+    """
+    for _ in range(max_retries):
+        try:
+            time.sleep(2)
```

Comment:
can we sleep for a random interval between 5 and 10 seconds rather than hardcoded 2 in case what is happening here is a result of multi-threaded tests all firing requests simultaneously?

Reply:
updated to sleep for 5-10 seconds